### DISCUSSION TITLE
Plot all gridlines

### DISCUSSION THREAD

**Discussion by bluetyson** [pyvista/pyvista#3763] [Q&A]
I don't think I can see how I can plot all the gridlines to match the ticks - it just does a default?

![Ruby Base Level 80](https://user-images.githubusercontent.com/17399794/210061452-b015379e-207d-43c6-bba4-ca0ccf13f76a.png)

e.g. if I want all the lines to show here - basically at 10m not 50m -- probably with thinner lines

**Comment by adeak** [comment]
Can you show how you created the plot? I suspect you'd have to use `show_grid(minor_ticks=True)`; is that what you used, giving you only minor ticks on the bounds but not on the grid?

**Comment by bluetyson** [comment]
Sure
```python
   text='Ruby Base Level ' + str(80-h)
    rplot = rubybase.clip_scalar(scalars='data', invert=True, value=818 - 18 - h*10).warp_by_scalar('data')
    rplot['elevation'] = rplot['data']
    plotter = pv.Plotter()
    plotter.add_title(text, font_size=18, color=None, font='arial', shadow=False)
    
    plotter.add_mesh(rplot, scalars='elevation',cmap='Blues_r',clim=818-18-h*10,opacity=0.5)
    plotter.show_bounds(rplot, minor_ticks=True,ticks='inside',grid='front')
    plotter.camera_position='xy'
```

**Reply by adeak** [reply]
My understanding is that `show_grid` is a thin wrapper to `show_bounds`, but the difference is exactly in the grid. So I'd recommend trying the same call but with `show_grid` instead of `show_bounds`. Does that help?

**Reply by bluetyson** [reply]
I don't think it did when I tried, but I will double check.

**Comment by bluetyson** [comment]
show_grid and minor ticks is the same I think?

**Comment by bluetyson** [comment]
```python
for h in range(2 ): #80
    text='Ruby Base Level ' + str(80-h)
    rplot = rubybase.clip_scalar(scalars='data', invert=True, value=818 - 18 - h*10).warp_by_scalar('data')
    rplot['elevation'] = rplot['data']
    plotter = pv.Plotter()
    plotter.add_title(text, font_size=18, color=None, font='arial', shadow=False)
    
    plotter.add_mesh(rplot, scalars='elevation',cmap='Blues_r',clim=818-18-h*10,opacity=0.5)
    plotter.show_grid(minor_ticks=True)
    plotter.camera_position='xy'
    plotter.screenshot(filename=r'D:\Dungeon23' + "\\" + text + '.png')
```
![Ruby Base Level 80](https://user-images.githubusercontent.com/17399794/210127346-4d69cb4e-524a-44a5-a942-ddf776863f57.png)

**Reply by bluetyson** [reply]
show_grid doesn't take the mesh as an argument as such, but the same
